js/signaling.js

```diff
@@ -186,9 +201,15 @@
 			return;
 		}
 
-		$.post(OC.generateUrl('/apps/spreed/signalling'), {
-			messages: JSON.stringify(this.spreedArrayConnection)
+		$.ajax({
+			url: OC.linkToOCS('apps/spreed/api/v1', 2) + 'signalling',
+			type: 'POST',
+			data: {messages: JSON.stringify(this.spreedArrayConnection)},
+			beforeSend: function (request) {
+				request.setRequestHeader('Accept', 'application/json');
+			}
 		});
+
 		this.spreedArrayConnection = [];
```

Comment:
I know that also was not handled before, but maybe we should only clear the list of pending messages if it could be sent, i.e. if an error occurred it should be retried on the next iteration.